components/ConvertForm/ManageConversion.js

```diff
@@ -74,6 +91,7 @@ function ManageConversion({ toAnj, fromAmount, toAmount, handleReturnHome }) {
         'Claim order',
         {
           createTx: () => claimOrder(openOrderHash, toAnj),
+          transactionComplete: hash => updateConvertedValue(hash),
```

Comment:
We may want to name this `transactionMined` instead; "complete" is a bit of a nebulous term for these transactions.